ibm_mq/datadog_checks/ibm_mq/metrics.py

```diff
@@ -100,13 +100,32 @@ def channel_status_metrics():
         'mca_status': pymqi.CMQCFC.MQIACH_MCA_STATUS,
         'msgs': pymqi.CMQCFC.MQIACH_MSGS,
         'ssl_key_resets': pymqi.CMQCFC.MQIACH_SSL_KEY_RESETS,
-        # NOTE: Following metrics are NOT tested in e2e. I didn't managed to to get those metrics locally.
         'batches': pymqi.CMQCFC.MQIACH_BATCHES,
         'current_msgs': pymqi.CMQCFC.MQIACH_CURRENT_MSGS,
         'indoubt_status': pymqi.CMQCFC.MQIACH_INDOUBT_STATUS,
     }
 
 
+def channel_stats_metrics():
+    return {
+        # Most stats metrics are count since we want to add up the values
+        # if there are multiple messages of same type for a single check run.
+        'msgs': (pymqi.CMQCFC.MQIAMO_MSGS, COUNT),
+        'bytes': (pymqi.CMQCFC.MQIAMO64_BYTES, COUNT),
+        'put_retries': (pymqi.CMQCFC.MQIAMO_PUT_RETRIES, COUNT),
+        # Following metrics are currently not covered by e2e tests
+        'full_batches': (pymqi.CMQCFC.MQIAMO_FULL_BATCHES, COUNT),
+        'incomplete_batches': (pymqi.CMQCFC.MQIAMO_INCOMPLETE_BATCHES, COUNT),
+        'avg_batch_size': (pymqi.CMQCFC.MQIAMO_AVG_BATCH_SIZE, GAUGE),
+    }
+
+
+def queue_stats_metrics():
+    return {
+        'q_min_depth': (pymqi.CMQCFC.MQIAMO_Q_MIN_DEPTH, GAUGE),
```

Comment:
note: there are more metrics we can collect, but this PR is large enough, it's better to do that in another PR.